code/datums/shuttles.dm

```diff
@@ -350,7 +353,7 @@
 	suffix = "raven"
 	name = "CentCom Raven Battlecruiser"
 	description = "The CentCom Raven Battlecruiser is currently docked at the CentCom ship bay awaiting a mission, this Battlecruiser has been reassigned as an emergency escape shuttle for currently unknown reasons. The CentCom Raven Battlecruiser should comfortably fit a medium to large crew size crew and is complete with all required facitlities including a top of the range CentCom Medical Bay."
-	admin_notes = "The long way home"
+	admin_notes = "Comes with turrets that will target any simplemob."
```

Comment:
oocky in my icky

Reply:
its an admin note you dip only admins see it

Reply:
:octocat:

Reply:
what did he mean by this

Reply:
🚲 🏠

Reply:
🔢 🥇